mlflow/R/mlflow/R/install.R

```diff
@@ -20,18 +20,26 @@ mlflow_maybe_create_conda_env <- function() {
 
 #' Install MLflow
 #'
-#' Installs auxiliary dependencies of MLflow (e.g. the MLflow CLI). As a one-time setup step, you
-#' must run install_mlflow() to install these dependencies before calling other MLflow APIs.
+#' Installs auxiliary dependencies of MLflow (e.g. the MLflow CLI). As a
+#' one-time setup step, you must run install_mlflow() to install these
+#' dependencies before calling other MLflow APIs.
 #'
 #' install_mlflow() requires Python and Conda to be installed.
 #' See \url{https://www.python.org/getit/} and \url{https://docs.conda.io/projects/conda/en/latest/user-guide/install/}.
 #'
+#' Alternatively, you can set MLFLOW_PYTHON_BIN and MLFLOW_BIN environment variables
```

Comment:
From offline discussion with @tomasatdatabricks, we're keeping this here instead of moving it after the examples as otherwise the build fails because it tries to interpret this comment as executable code